signing_service/signing_service/tests/test_signing_service_methods.py

```diff
@@ -208,7 +209,7 @@ def test_that_argument_parser_should_parse_correct_input_and_use_default_values(
 
     def test_that_argument_parser_should_fail_if_port_cannot_be_casted_to_int(self):
         sys.argv += [
-            '--concent_cluster_host', '127.0.0.1',
+            '--concent-cluster-host', '127.0.0.1',
```

Comment:
As far I know it should be also changed in deployment? @bartoszbetka @cameel

Reply:
No no, I have only fixed this name in this tests, because it have old version, but the test still works because the result was `SystemExit` too.

Reply:
Ooooh, I see. My bad. Approve :P